nototools/shape_diff.py

```diff
@@ -116,6 +118,12 @@ def find_rendered_diffs(self, font_size=24, stats=None):
                 'hb-view', '--font-size=%d' % font_size,
                 '--output-file=%s' % b_png,
                 '--features=%s' % ','.join(features), path_b, text])
+
+            img_info = subprocess.check_output(['identify', a_png]).split()
+            assert img_info[0] == a_png and img_info[1] == 'PNG'
+            subprocess.call([
+                'convert', '-gravity', 'center', '-background', 'black',
+                '-extent', img_info[2], b_png, b_png])
             with open(diffs_filename, 'a') as ofile:
                 subprocess.call(
                     ['compare', '-metric', 'AE', a_png, b_png, cmp_png],
```

Comment:
I'll use that for actually saving comparison images; this is just used for getting the absolute error metric.